core/domain/playthrough_issue_registry_test.py

```diff
@@ -18,14 +18,53 @@
 
 from __future__ import annotations
 
+from core import utils
 from core.domain import playthrough_issue_registry
 from core.tests import test_utils
+from extensions.issues.CyclicStateTransitions import CyclicStateTransitions
+from extensions.issues.EarlyQuit import EarlyQuit
+from extensions.issues.MultipleIncorrectSubmissions import (
+    MultipleIncorrectSubmissions)
 
 
 class IssueRegistryUnitTests(test_utils.GenericTestBase):
     """Test for the issue registry."""
 
+    def setUp(self):
+        super(IssueRegistryUnitTests, self).setUp()
+        self.issues_dict = {
+                'EarlyQuit':
```

Comment:
The indentation from the previous line should only be by 4.

Don't have line breaks after the colons; put the key and value in a single line. If you need to break the lines due to them being too long, do something like this:

```
'CyclicStateTransitions': (
    CyclicStateTransitions.CyclicStateTransitions)
```

The final } should be at the same indentation level as the line containing the {.

Reply:
Thanks Sean !! I will implement that asap :)